packages/text/src/Text.ts

```diff
@@ -394,6 +394,9 @@ export class Text extends Sprite
 
         baseTexture.setRealSize(canvas.width, canvas.height, this._resolution);
 
+        // Recursively updates transform of all objects from the root to this one
+        (this as any)._recursivePostUpdateTransform();
```

Comment:
Much better fix, however, can you please remove `as any`. Keeping it like this masks these API when we have to refactor things later.

The approach to deal with private APIs that are used internally, is to remove the access modifier (in this case `private`) from the original method. We can use documentation flagging to make sure it's not public. 

https://github.com/pixijs/pixi.js/blob/3f90364d02b11b4fef1645df2bedac3e9b2b121e/packages/display/src/DisplayObject.ts#L283

Reply:
yes,. this really should not be private